docs/source/user_data/create-datasets-for-dashboard.ipynb

```diff
@@ -140,15 +261,72 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%bash -s \"$location\"\n",
-    "rio bounds $1 | supermercado burn 1 # this last value is the \"zoom\""
+    "%%capture\n",
+    "!pip install cogeo-mosaic"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "import os\n",
+    "import urllib\n",
+    "\n",
+    "from cogeo_mosaic.mosaic import MosaicJSON\n",
+    "from cogeo_mosaic.backends import MosaicBackend"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Skipping this step since we don't want to upload these large files more than once!\n",
+    "full_path_files = [f'{project_dir}/{file}' for file in files]\n",
+    "# maap.uploadFiles(full_path_files)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "uuid = 'xxx'\n",
+    "tiles = [generate_s3_upload_path(uuid, file) for file in files]\n",
+    "print(tiles)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mosaicdata = MosaicJSON.from_urls(tiles, minzoom=1, maxzoom=16)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Test the link with the titiler"
+    "## Using mosaicJSON with titiler\n",
+    "\n",
+    "There are 2 options for using mosaicJSON with titiler:\n",
+    "\n",
+    "1. (Preferred) Post mosaicJSON to titiler `mosaics` endpoint and use the `mosaicjson/mosaics` endpoint for dynamic tiling.\n",
+    "2. Upload mosaicJSON to S3 and pass the S3 url to the titiler `tiles` endpoint."
```

Comment:
I find `tiles` confusing in this context, `mosaicjson/tiles` might be more clear